silkworm/rpc/json/execution_payload.cpp

```diff
@@ -88,7 +97,20 @@ void from_json(const nlohmann::json& json, ExecutionPayload& execution_payload)
         .logs_bloom = logs_bloom,
         .extra_data = *silkworm::from_hex(json.at("extraData").get<std::string>()),
         .transactions = transactions,
-        .withdrawals = withdrawals};
+        .withdrawals = std::move(withdrawals),
+        .blob_gas_used = blob_gas_used,
+        .excess_blob_gas = excess_blob_gas,
+    };
+
+    // Set the ExecutionPayload version (default is V1)
+    SILKWORM_ASSERT(execution_payload.version == ExecutionPayload::V1);
+    if (execution_payload.withdrawals) {
+        if (execution_payload.blob_gas_used && execution_payload.excess_blob_gas) {
```

Comment:
Both fields are optional so probably the condition is more like
```suggestion
        if (execution_payload.blob_gas_used || execution_payload.excess_blob_gas) {
```